3rdparty/expreval/docs/changes.txt

```diff
@@ -2,6 +2,7 @@ IFEM Hacks
 ----------
 - Added a define in defs.h. auto_ptr is deprecated in c++0x. uses
 std::unique_ptr instead. Quells the warnings
+- Added integration with autodiff
```

Comment:
Since you took the hint to update this change log, maybe make it complete. A few of the other changes from the commit log could be mentioned here (pow function, support for exponent constants, exception handling improvement, ...)

Reply:
okay.